logger/build.gradle

```diff
@@ -3,8 +3,9 @@ apply plugin: 'com.palantir.revapi'
 
 dependencies {
     api project(':safe-logging')
-    api 'com.google.errorprone:error_prone_annotations'
     implementation project(':logger-spi')
+    implementation 'com.google.errorprone:error_prone_annotations'
```

Comment:
updated this from api to implementation to avoid leaking deps into consumer projects.